torch/onnx/symbolic_opset9.py

```diff
@@ -2622,13 +2622,17 @@ def is_floating_point(g, self):
     return g.op("Constant", value_t=torch.BoolTensor([0]))
 
 
-def __isnot_(g, self, other):
+def __is_(g, self, other):
```

Comment:
what's the significance of the trailing underscore?
A trailing underscore has a special meaning in PyTorch that it's an in-place op, but does that make sense here?
Can we just remove the trailing underscores?

Reply:
Looks like '__is_' is the name that the IR emitter assigns. Most of the logical ops representing Tree kinds have the similar pattern:
https://github.com/pytorch/pytorch/blob/5060b69d62943e11f053f8c86b8154a7d009af11/torch/csrc/jit/frontend/ir_emitter.cpp#L2891
Not sure about the logic, but I guess changing the pattern would be a significant update to the torch IR.